command/v6/unset_env_command.go

```diff
@@ -2,12 +2,14 @@ package v6
 
 import (
 	"code.cloudfoundry.org/cli/command"
+	"code.cloudfoundry.org/cli/command/flag"
 	"code.cloudfoundry.org/cli/command/translatableerror"
 )
 
 type UnsetEnvCommand struct {
-	usage           interface{} `usage:"CF_NAME unset-env APP_NAME ENV_VAR_NAME"`
-	relatedCommands interface{} `related_commands:"apps, env, restart, set-staging-environment-variable-group, set-running-environment-variable-group"`
+	RequiredArgs    flag.UnsetEnvironmentArgs `positional-args:"yes"`
```

Comment:
The `RequiredArgs` is not needed for this pull request but we can add it once we refactor the command.